### PR TITLE
OE-374, fix the audit page be empty error for most users

### PR DESCRIPTION
**Pull request by Zhaolian-Zhou**
1. Remove the hardcoded userid in beforAction() function, 
2. Change access rule from "OprnViewClinical" to "admin"

### CODE REVIEW

**Review by apigram — COMMENTED** [review]
The same rule will need to be applied to the audit menu bar item in the config files as well. That way, users without the correct role will not be able to access the screen through the frontend.

**Comment by Zhaolian-Zhou** [comment]
It is restricted to "admin" already in the config files.

**Comment by apigram** [comment]
It seems you are indeed correct. I'll approve this pull request.

**Review by apigram — APPROVED** [review]

File: protected/controllers/AuditController.php [hunk 1]
```diff
@@ -28,18 +28,13 @@ public function accessRules()
...
+                'roles' => array('admin'),
             ),
         );
     }
 
     public function beforeAction($action)
     {
-        Yii::app()->assetManager->registerScriptFile('js/audit.js');
-        $userid = Yii::app()->session['user']->id;
-        if (($userid != 2103) and ($userid != 122) and ($userid != 613) and ($userid != 1330) and ($userid != 1)) {
```
Comment (marshl) [inline protected/controllers/AuditController.php]: Curiously, this same logic is also in DicomLogViewerController, but commented out. It even uses teh same user IDs

**Pull request merged**